src/daemon/db_util.rs

```diff
@@ -142,6 +149,96 @@ async fn transcode_into_forest_car(from: &Path, to: &Path) -> anyhow::Result<()>
     Ok(())
 }
 
+pub fn populate_eth_mappings<DB>(
```

Comment:
Could you describe what we're mapping and how many mappings we expect in a comment here?

Reply:
Sure.